Prevent: Variable MPI_HOME="" does not seem to contain a valid mpicc.
Author: Fokke Dijkstra (University of Groningen)
--- src/make_mpich	2015-02-16 13:31:11.838082386 +0100
+++ src/make_mpich	2015-02-16 13:40:19.000844455 +0100
@@ -1,5 +1,5 @@
 # Enter root directory of mpich install
-MPI_HOME=
+#MPI_HOME=
 
 MPICC=$(shell find ${MPI_HOME} -name mpicc -print)
 
